pkg/api/volume_req.go

```diff
@@ -30,6 +30,13 @@ type VolCreateReq struct {
 	Experimental bool              `json:"experimental,omitempty"`
 	Deprecated   bool              `json:"deprecated,omitempty"`
 	Metadata     map[string]string `json:"metadata,omitempty"`
+	/*supported Flags
```

Comment:
I'd prefer this to be at the top of this structure and not in the middle. You can host `godoc` locally to check rendering.
Same below.

Reply:
ran godoc locally ,it is rendering properly

Reply:
it is rendering properly in local godoc